src/FSharpFakeTargets/targets.fs

```diff
@@ -31,6 +32,8 @@ module Targets =
       Publish:                 bool
       AccessKey:               string
       PublishUrl:              string
+      Properties:              List<string * string>
```

Comment:
For consistency with the other keys in this record, could we do `(string * string) list`